Add contributing instructions to readme (#342)
unitaryfund/unitary.fund: README.md

[![Unitary Fund](https://img.shields.io/badge/Supported%20By-UNITARY%20FUND-brightgreen.svg?style=for-the-badge)](https://unitary.fund)
```

## Writing a blog post 
To add a blog post, you just need to add a markdown file (`YYYY_your_title.md`) under the `src/content/blog/` folder. 

> An example file is this [one on 2023 UF members](https://raw.githubusercontent.com/unitaryfund/unitary.fund/main/src/content/blog/2023_members.md), which renders [here](https://unitary.fund/posts/2023_members/). Make sure to edit the keys correctly, e.g., below
> ```
> ---
> title: Announcing the 2023 Unitary Fund Members
> author: Unitary Fund Team
> day: 7
> month: 2
> year: 2023
> ---
> 
> Since its launch last year...
> ```

Reviewer: Two things:

1. Do we need to use the full GitHub link? I think a local reference might work (and is more convenient for local IDE development).
2. What do you think about making this a generic template?

````suggestion
An example file is this [one on 2023 UF members](./src/content/blog/2023_members.md), which renders [here](https://unitary.fund/posts/2023_members/).
The following template post can be used to start from scratch:
```
---
title: post title
author: author one and author two
day: 7
month: 2
year: 2023
---

your markdown content
```
````

> Once you open a pull request (PR) with edits, you get to see the preview via the Vercel automation. If you click on the "Visit preview" link that shows up in the bot comment, you can see how the edits will look like once the PR is merged. If they look good, you can ask for a review by calling one of the UF team members (e.g., @nathanshammah). 

Reviewer: ```suggestion
After opening a pull request (PR), the Vercel automation will create a preview you can access by clicking the "Visit preview" link that shows up in the bot comment. If the changes look good, request a review by tagging one of the UF team members (e.g., @nathanshammah). 
```

### Adding images to a blog post

> To add images, you can include in the website source under `public/images/` folder, and embed them in a page using the markdown syntax (e.g., `![](/images/your_image.png)`) markdown file like this one). 

Reviewer: ```suggestion
To add images, add the image file under the `public/images/` folder, and embed it in the markdown page using the `![](/images/your_image.png)` syntax.
```

## Adding a new grant 
To add a new grant, you just need to add a markdown file (`YYYY_project_name.md`) under the `src/content/grant/` folder. 

> An example file is this [2023 grant to TorchQuantum](https://raw.githubusercontent.com/unitaryfund/unitary.fund/main/src/content/grant/2023_TorchQuantum.md). Make sure to edit the keys correctly, e.g. use [alpha-2 code](https://www.iban.com/country-codes) for the `country` key, as shown below

Reviewer: Good thinking with the county codes link.
